tests/unit/amo/components/TestSearchResult.js

```diff
@@ -125,53 +123,6 @@ describe(__filename, () => {
     );
   });
 
-  it('renders the user count', () => {
-    const root = render({ addon: { ...baseAddon, average_daily_users: 6233 } });
-
-    expect(root.find('.SearchResult-users')).toIncludeText('6,233 users');
-  });
-
-  it('localises the user count', () => {
```

Comment:
Why is this test case deleted?

Reply:
I encountered the following error while running the test.

```
 tests/unit/amo/components/TestSearchResult.js
  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestSearchResult.js › renders the user count

    toIncludeText must be called on a single node, not an empty node.

      80 |     const root = render({ addon: { ...baseAddon, average_daily_users: 6233 } });
      81 | 
    > 82 |     expect(root.find('.SearchResult-users')).toIncludeText('6,233 users');
         |                                              ^
      83 |   });
      84 | 
      85 |   it('localises the user count', () => {

      at Object.<anonymous> (tests/unit/amo/components/TestSearchResult.js:82:46)
```
I am investigating it right now.

Reply:
We need this test case back. I'll add it.